README.md

```diff
@@ -26,7 +26,22 @@
           "app_category_type": "public.app-category.productivity"
         },
         "linux": {
-          "icon": "resources/linux/<your-project-id>.png"
+          "icon": "resources/linux/<your-project-id>.png",
+          "debian_config": {
```

Comment:
@rueckstiess Suggested `debianConfig` which was inconsistent with `app_category_type`, @rueckstiess were you strongly opinionated on `debian` vs `debian_config`?